foundations/html_css/intro-to-css.md

```diff
@@ -31,52 +32,52 @@ Selectors simply refer to the HTML elements to which CSS rules apply; they're wh
 
 The universal selector will select elements of any type, hence the name "universal", and the syntax for it is a simple asterisk. In the example below, every element would have the `color: purple;` style applied to it.
 
-~~~css
+```css
```

Comment:
Nit: Sorry, just noticing this. The markdown parser we use on the site needs multiline code blocks like this to be started and ended with three tildes(~~~) instead of back ticks(```)

Example:
```suggestion
~~~css
* {
  color: purple;
}
~~~
```
Can all the changed code blocks be reverted back to tildes please?

Reply:
I think I've resolved it. Thank you for the detailed explanation :)

Reply:
Fixing the merge conflict reverted it back to backticks again. Apologies @JaeGif, I should have left that feedback until after the conflict was fixed so not to confuse things.